### PR TITLE
msgfmt, msgunfmt: add page

### PR DESCRIPTION
**Pull request by wtflm**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/msgfmt.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# msgfmt
+
+> Compile message catalog to binary format.
+> More information: <https://www.gnu.org/software/gettext/manual/html_node/msgfmt-Invocation.html>.
+
+- Convert a .po file to an .mo file:
```
[the remark this answers is not in the document]
Comment (wtflm) [reply pages/common/msgfmt.md:6]: Hmm, I guess I read it as "em-oh" rather than "mo" while skipping "dot".

File: pages/common/msgunfmt.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# msgunfmt
+
+> Decompile message catalog binary format.
+> More information: <https://www.gnu.org/software/gettext/manual/html_node/msgunfmt-Invocation.html>.
+
+- Output conversion:
+
+`msgunfmt {{path/to/file.mo}}`
+
+- Convert a .mo file to a .po file:
```
Comment (kbdharun) [inline pages/common/msgunfmt.md:10]:
```suggestion
> Decompile message catalog from the binary format.
> More information: <https://www.gnu.org/software/gettext/manual/html_node/msgunfmt-Invocation.html>.

- Output conversion:

`msgunfmt {{path/to/file.mo}}`

- Convert a `.mo` file to a `.po` file:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**